pkg/operator/configobservation/images/observe_images.go

```diff
@@ -30,14 +30,16 @@ func ObserveInternalRegistryHostname(genericListers configobserver.Listers, exis
 	observedConfig := map[string]interface{}{}
 	configImage, err := listers.ImageConfigLister.Get("cluster")
 	if errors.IsNotFound(err) {
-		glog.Warningf("image.config.openshift.io/cluster: not found")
+		glog.V(4).Infof("image.config.openshift.io/cluster: not found")
 		return observedConfig, errs
 	}
 	if err != nil {
+		glog.Warningf("error getting image.config.openshift.io/cluster: %v", err)
 		return prevObservedConfig, errs
 	}
 	internalRegistryHostName := configImage.Status.InternalRegistryHostname
 	if len(internalRegistryHostName) > 0 {
+		glog.V(4).Infof("setting internal registry hostname to: %q", internalRegistryHostName)
```

Comment:
shouldn't this be differential? We don't want an info (even verbosity 4) if nothing changed.

Reply:
yeah, I think we want info when things change... I can compare current value (we should do that anyway)